.husky/pre-commit

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+RED="\033[1;31m"
+GREEN="\033[1;32m"
+NC="\033[0m"
+linter_exit_code=1
+staged_js_files=$(git diff --cached --diff-filter=d --name-only | grep .js$)
+./node_modules/.bin/eslint $staged_js_files --quiet --fix
+linter_exit_code=$?
+git add -f $staged_js_files
+if [ $linter_exit_code -ne 0 ]
+then
+  echo "${RED} ❌ Linter errors have occurred ${NC}"
```

Comment:
You will need to pass the `-e` flag otherwise the backslashes in the colour code won't be interpreted.

Reply:
@algomaster99 
![Screenshot from 2021-05-03 21-46-49](https://user-images.githubusercontent.com/59891164/116902872-513dbf00-ac59-11eb-936c-d90d7bb99fb0.png)

When run on Linux the command is showing proper colors as shown in the above image. 
The repo is different which was used for testing purposes.

Reply:
That is weird. I had to pass the `-e` flag on windows git bash.

Reply:
@algomaster99 
I think it may vary according to the different operating systems. On my git bash(opened via vscode), it didn't give the color even when I change to `-e` flag.

Reply:
You have git bash installed on Windows?

Pass it as `echo -e ...`.

Reply:
@algomaster99 
I passed the `-e` flag in the wrong way. I have updated my PR also following the same. Let me know if there are any changes required.